tests/materialize/materialize-databricks/snapshot.json

```diff
@@ -406,38 +205,38 @@
 {
   "rows": [
     {
-      "flow_document": "{\"_meta\":{\"uuid\":\"8de85150-20e1-11ee-990b-ffd12dfcd47f\"},\"id\":1,\"int\":6,\"str\":\"str 6\"}",
-      "flow_published_at": "2023-07-12T18:26:01.560712Z",
+      "flow_document": "{\"_meta\":{\"uuid\":\"77a80200-1dda-11b2-8000-071353030311\"},\"id\":1,\"int\":7,\"str\":\"str 6\"}",
+      "flow_published_at": "1970-01-01T01:00:04Z",
       "id": 1,
-      "int": 6,
+      "int": 7,
       "str": "str 6"
     },
     {
-      "flow_document": "{\"_meta\":{\"uuid\":\"957348bc-20e1-11ee-990b-ffd12dfcd47f\"},\"id\":2,\"int\":7,\"str\":\"str 7\"}",
-      "flow_published_at": "2023-07-12T18:26:14.215494Z",
+      "flow_document": "{\"_meta\":{\"uuid\":\"78409880-1dda-11b2-8000-071353030311\"},\"id\":2,\"int\":9,\"str\":\"str 7\"}",
+      "flow_published_at": "1970-01-01T01:00:05Z",
       "id": 2,
-      "int": 7,
+      "int": 9,
       "str": "str 7"
     },
     {
-      "flow_document": "{\"_meta\":{\"uuid\":\"9afb3ff6-20e1-11ee-990b-ffd12dfcd47f\"},\"id\":3,\"int\":8,\"str\":\"str 8\"}",
-      "flow_published_at": "2023-07-12T18:26:23.495167Z",
+      "flow_document": "{\"_meta\":{\"uuid\":\"78d92f00-1dda-11b2-8000-071353030311\"},\"id\":3,\"int\":11,\"str\":\"str 8\"}",
+      "flow_published_at": "1970-01-01T01:00:06Z",
       "id": 3,
-      "int": 8,
+      "int": 11,
```

Comment:
These differences in the `int` values is due to a `sum` reduction strategy which was not working previously when using `materialize-fixtures`, but it works with `flowctl preview`